### PR TITLE
Make seekbar snapping range configurable

### PR DESCRIPTION
**Pull request by aygee**
I would like to be able to control the snapping range, rather than the preset 1%. In my use case, I would like to disable it by setting it to 0.

### CODE REVIEW

File: src/ts/components/seekbar.ts [hunk 1]
```diff
@@ -253,6 +259,10 @@ export class SeekBar extends Component<SeekBarConfig> {
       if (this.isUiShown) {
         this.setAriaSliderValues();
       }
+
+      if (typeof uimanager.getConfig().seekbarSnappingRange !== 'undefined') {
```
Comment (bitmovin-kenny) [inline src/ts/components/seekbar.ts:263]:
```suggestion
      if (typeof uimanager.getConfig().seekbarSnappingRange === 'number') {
```
would make more sense IMO

**Comment by aygee** [comment]
> Please add an entry to CHANGELOG.md

@bitmovin-kenny What version should I add it to? v3.14.0?

**Comment by bitmovin-kenny** [comment]
We collect new features in a `development` section until a new version is released. So something like this:

```md
## [development]

### Added
- Seekbar snapping range is now configurable
```

**Comment by aygee** [comment]
@bitmovin-kenny Done. Thanks.

File: src/ts/components/seekbar.ts [hunk 2]
```diff
@@ -253,6 +259,10 @@ export class SeekBar extends Component<SeekBarConfig> {
       if (this.isUiShown) {
         this.setAriaSliderValues();
       }
+
+      if (typeof uimanager.getConfig().seekbarSnappingRange === 'number') {
+        this.config.snappingRange = uimanager.getConfig().seekbarSnappingRange;
+      }
```
Comment (bitmovin-kenny) [inline src/ts/components/seekbar.ts:265]: I didn't see it the first time, but we should not need to do this on every playback position update.

I would suggest moving it to after the marker config initializiation in line 389:

```
    // Add markers when a source is loaded or update when a marker is added or removed
    uimanager.getConfig().events.onUpdated.subscribe(() => {
      playbackPositionHandler();
    });
```

Comment (aygee) [reply src/ts/components/seekbar.ts]: Good point, I didn't realise that. I've moved the code. Thanks!

**Comment by aygee** [comment]
@bitmovin-kenny I've created a new PR to develop on #302. Closing this PR.

**Pull request closed without merging**